packages/did/src/DidDetails/FullDidDetails.ts

```diff
@@ -40,6 +40,7 @@ const methodMapping: Record<string, VerificationKeyRelationship | undefined> = {
   'did.submitDidCall': undefined,
   didLookup: 'authentication',
   publicCredentials: 'assertionMethod',
+  dipProvider: 'authentication',
```

Comment:
unrelated, but do you think this configuration could be moved into runtime constants at some point, either on the pallet level or in the did pallet?

Reply:
You mean exposed as part of the metadata?

Reply:
Yeah that's probably how these are exposed then. If we found a solution for this chain-side, we wouldn't have to duplicate it client-side, making the logic so much more versatile

Reply:
I mean, in this case I am not sure it is worth the roundtrip and load on a fullnode to derive this information. Better would be to have a WASM blob, which we tried and failed earlier this year.

Reply:
Unless we can expose it in a constant way via metadata, but I am not sure that is possible. Will make a ticket to investigate that at some point.

Reply:
Yes, check it out, but I would guess that the constant values are exposed via the metadata and thus available as soon as the metadata has been loaded - I don't think this puts any additional load on the node